hare/hare.go

```diff
@@ -206,6 +206,9 @@ func (h *Hare) onTick(id types.LayerID) {
 		return
 	}
 
+	// call to start the calculation of active set size beforehand
+	go h.rolacle.IsIdentityActiveOnConsensusView(h.nid.Key, id)
```

Comment:
what does this has to do with metrics ?

Reply:
shaatnez shel barak

Reply:
I'd still rather you put it in a different pr